packages/abap-deploy-config-inquirer/src/utils.ts

```diff
@@ -175,16 +175,18 @@ export function getPackageAnswer(previousAnswers?: AbapDeployConfigAnswersIntern
  * Determines the transport request from the various transport related prompts.
  *
  * @param promptAnswers - previous answers
+ * @param transportOpt - transport answer passed as an option e.g. from headless
  * @returns transport request
  */
-export function getTransportAnswer(promptAnswers?: AbapDeployConfigAnswersInternal): string {
+export function getTransportAnswer(promptAnswers?: AbapDeployConfigAnswersInternal, transportOpt?: string): string {
```

Comment:
On the face of it, passing this additional arg seems pointless as its immediately returned. Consider not calling this function instead?

Reply:
Thanks, yeah makes sense. I've updated